### PR TITLE
release v1.2.0

### PR DESCRIPTION
**Pull request by estroz**
**Description of the change:** release commit

**Motivation for the change:** release v1.2.0

**Checklist**

If the pull request includes user-facing changes, extra documentation is required:
- [ ] Add a new changelog fragment in `changelog/fragments` (see [`changelog/fragments/00-template.yaml`](https://github.com/operator-framework/operator-sdk/tree/master/changelog/fragments/00-template.yaml))
- [ ] Add or update relevant sections of the docs website in [`website/content/en/docs`](https://github.com/operator-framework/operator-sdk/tree/master/website/content/en/docs)

### CODE REVIEW

**Comment by asmacdo** [comment]
The 2 external link failures are: 
1. Netlify subdomain mapping. I set it up to watch for the v1.2.x branch ahead of time but after that branch is created someone needs to go into netlify and add the subdomain. (I've just done this, so the v1-2-x.sdk.operatorframework.io link now works.)
2. The other broken link is this bug https://github.com/operator-framework/operator-sdk/issues/4157 There is already a fix for it that I thought would merge before the release: https://github.com/operator-framework/operator-sdk/pull/4156/files#diff-38c47b01ea565a10189c9ef7211e75e02e5072e0bfc16f09469ac4aa96d592aaR15  IMO, just add the check links diff to this commit.

**Review by jmrodri — APPROVED** [review]
/lgtm

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,31 @@
...
+
+### Changes
+
+- Improved `ansible-operator` and `helm-operator` help text. ([#4187](https://github.com/operator-framework/operator-sdk/pull/4187))
+- When generating bundles and packagemanifests, add all resources supported by OLM. ([#4137](https://github.com/operator-framework/operator-sdk/pull/4137))
+- `olm` and `run` subcommands will print aggregated resource errors when either OLM or an operator fail to install, respectively. ([#3787](https://github.com/operator-framework/operator-sdk/pull/3787))
+- Upgrade sigs.k8s.io/controller-runtime dependency version from `v0.6.2` to [`v0.6.3`](https://github.com/kubernetes-sigs/controller-runtime/releases/tag/v0.6.3). ([#4062](https://github.com/operator-framework/operator-sdk/pull/4062))
+- For Golang based operators with multigroup support, fix scaffolded `controllers/<group>/suite_test.go` to have a correct `envtest.Environment.CRDDirectoryPaths` ([kubebuilder#1665](https://github.com/kubernetes-sigs/kubebuilder/pull/1665)). ([#4062](https://github.com/operator-framework/operator-sdk/pull/4062))
```
Comment (camilamacedo86) [inline CHANGELOG.md:13]: This one should be in the bug fixes. Could you please move it before get the PR merge?

**Review by camilamacedo86 — APPROVED** [review]
/lgtm

**Pull request merged**